DXRModules/DeusEx/Classes/DXRFixup.uc

```diff
@@ -287,6 +308,13 @@ function PostFirstEntryMapFixes()
         break;
 
 #ifndef revision
+    case "06_HONGKONG_VERSALIFE":
+        foreach AllActors(class'Male1',male){
+            if (male.BindName=="Disgruntled_Guy"){
+                male.bImportant=True;
```

Comment:
What if he's already dead when we first enter this map? I think it's technically possible especially with entrance rando, and this might happen after the ScriptedPawn checks its own _Dead flag